### PR TITLE
[SW-2210]wrong date content collection

### PR DESCRIPTION
**Pull request by yeniatencio**
## Motivation and Context

**JIRA issue:**https://digital-vic.atlassian.net/browse/SW-2210

## Changed

1. Adding field_news_date for content type News for the search results. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

File: packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js [hunk 1]
```diff
@@ -992,13 +992,13 @@ module.exports = class ContentCollection {
...
     const link = this.getLocalisedLinkFromSource(_source)
-
+    const newsDate = _source.field_news_date
     switch (this.getDisplayResultComponentType()) {
       case 'search-result':
         mappedResult = {
           title: _source.title?.[0],
           link: link ? { linkText: link.domain + link.path, linkUrl: link.path } : null,
-          date: _source.created?.[0],
+          date: newsDate ? _source.field_news_date?.[0] : _source.created?.[0],
```
Comment (waitingallday) [inline packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js:1001]: You're defining `newsDate` above, so you can use it here as well eg.
```
date: newsDate ? newsDate[0] : _source.created?.[0],
```
Otherwise you could drop that const entirely:
```
date: _source.field_news_date ? _source.field_news_date[0] : _source.created?.[0],
```

File: examples/reference/.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Tide API
-CONTENT_API_SERVER=https://develop.content.reference.sdp.vic.gov.au/
+CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au
```
Comment (waitingallday) [inline examples/reference/.env:2]: Can you revert this change back to develop.content

Comment (dylankelly) [inline examples/reference/.env:2]: @yeniatencio please revert this change.

The better way to change the CONTENT_API_SERVER value for a PR branch is to use the lagoon CLI tool to change the value, that way you don't need to commit this file. 

See https://digital-vic.atlassian.net/wiki/spaces/DDS/pages/2050949826/How-to+update+an+environment+variable+using+Lagoon

Otherwise if its just for local development just either run the dev command with the value before it eg: `CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au npm run dev` _OR_ export the value, eg: `export CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au`

File: packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js (hunk 1 above)
Comment (dylankelly) [inline packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js:1001]: This doesn't read very well as is, instead of a ternary here this could be a [logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) and use [optional chaining ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to guard against undefined values  

eg 
```
date: _source.field_news_date?.[0] || _source.created?.[0]
```
This reads as:
if `field_news_date` is defined and the first item in the array is not falsy, use that otherwise use the other value

Even better would be to [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#binding_and_assignment) `_source` for just the values needed, but thats unrelated to this PR.

**Review by dylankelly — CHANGES_REQUESTED** [review]

File: packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js (hunk 1 above)
Comment (yeniatencio) [reply packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js]: Thank you @dylankelly. I have updated my pr for your review.

File: examples/reference/.env (hunk 2 above)
Comment (yeniatencio) [reply examples/reference/.env]: Done @waitingallday Ta

**Review by dylankelly — APPROVED** [review]

**Review by ChrisCampbellCodes — APPROVED** [review]
